src/main/webapp/new/app/shared/domain/game-map/edge.ts

```diff
@@ -80,6 +80,17 @@ export class Edge {
         return null;
     }
 
+    isJoint() {
+        let hexesCount = 0;
+        for (let i in this.hexes) {
+            hexesCount++;
+            if (hexesCount > 1) {
+                return true;
+            }
+        }
+        return false;
+    }
```

Comment:
maybe we can replace this method with something like this: 
`return this.hexes.size > 0`